runtime/runtime-params-estimator/src/gas_metering.rs

```diff
@@ -175,10 +175,13 @@ pub fn compute_gas_metering_cost(
     let store = create_store(&get_store_path(workdir.path()));
     let cache_store = Arc::new(StoreCompiledContractCache { store });
     let cache: Option<&dyn CompiledContractCache> = Some(cache_store.as_ref());
-    let vm_config_gas = VMConfig::default();
+    let protocol_version = ProtocolVersion::MAX;
```

Comment:
add a comment why we're using protocolversion::MAX here

Reply:
Yeah, this actually should be `PROTOCOL_VERSION` I believe. `ProtocolVersion::MAX` sadly looks like something reasonable, but it is not. `ProtocolVersion` is just `u64`, so that is just `!0` bit pattern, and not some logically meaningful protocol version. `PROTOCOL_VERSION` is the constant defined  by us, which means the latest protocol version supported.